worlds/alttp/__init__.py

```diff
@@ -544,6 +544,44 @@ def stage_extend_hint_information(cls, world, hint_data: typing.Dict[int, typing
                             er_hint_data[region.player][location.address] = main_entrance.name
         hint_data.update(er_hint_data)
 
+    @classmethod
+    def stage_modify_multidata(cls, multiworld, multidata: dict):
+
+        ordered_areas = (
+            'Light World', 'Dark World', 'Hyrule Castle', 'Agahnims Tower', 'Eastern Palace', 'Desert Palace',
+            'Tower of Hera', 'Palace of Darkness', 'Swamp Palace', 'Skull Woods', 'Thieves Town', 'Ice Palace',
+            'Misery Mire', 'Turtle Rock', 'Ganons Tower', "Total"
+        )
```

Comment:
this looks like the list we have in main.py?

Reply:
it should be removed from main.py with this though?

Reply:
yeah, it is. i mixed up stuff